.github/workflows/release.yaml

```
@@ -3,16 +3,48 @@ on:
   push:
     tags:
       - v*
-
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
 jobs:
   dist:
     runs-on: ubuntu-latest
+    needs:
+      - lint
+      - test
+      - build
+      - e2e
+      - docker
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: "0"
 
-      # TODO: Add build steps here
+      - uses: actions/download-artifact@v2
+        with:
+          name: dist
+          path: dist
```

I'm not sure if that actually works and if that downloads the correct artifact. I would be fine if we do a build again,
e.g.
```suggestion
      - uses: actions/setup-node@v2
        with:
          node-version: '16'
          cache: 'npm'
      - run: npm run build
```
Assuming that is, if builds are reasonably reproducible with package-lock.json

---

That makes sense. I replaced the download action with the build steps.